com.github.lruzicka.Needly.yaml

```diff
@@ -0,0 +1,113 @@
+# The YAML manifest for the Needle Flatpak
+#
+app-id: com.github.lruzicka.Needly
+runtime: org.freedesktop.Platform
+runtime-version: '21.08'
+sdk: org.freedesktop.Sdk
+command: needly
+finish-args: 
+   - --share=ipc # For X11
+   - --socket=wayland # Enable communication with Wayland
+   - --socket=x11 # Alternatively, with X11
+   - --device=kvm # Allow access to a KVM device
```

Comment:
I must misunderstand what the app does to ask why it wants `kvm` access

Reply:
I think it needs it to be able to take the screenshots from a running VM. I can however try and compile without this flag, if it still works.

Reply:
The app does not need the `kvm` access to do what it should. I have removed it.